src/NLog/Targets/MethodCallParameter.cs

```diff
@@ -111,5 +111,25 @@ public MethodCallParameter(string name, Layout layout, Type type)
         /// <docgen category='Parameter Options' order='10' />
         [RequiredParameter]
         public Layout Layout { get; set; }
+
+        /// <summary>
+        /// Parameter can combine multiple LogEvents into a single parameter value
+        /// </summary>
+        public bool GroupLayoutActive { get; set; }
```

Comment:
AFAIK the most targets in NLog uses the `Enable...` names.

So EnableGrouping?